src/nvim/api/extmark.c

```diff
@@ -559,6 +561,8 @@ Integer nvim_buf_set_extmark(Buffer buffer, Integer ns_id, Integer line, Integer
     { NULL, NULL, NULL },
   };
 
+  // uncrustify:on
```

Comment:
might be clearer if this sticks next to the relevant code, i.e. no blank line. same for the start.